### PR TITLE
Detect new brands:  8848, Transpeed and Detect devices for existing brands: ZTE, Realme, Oppo, Vivo, Meitu, Chuwi, Motorola, Asus, Sony, Huawei, Xiaomi

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -7063,11 +7119,13 @@ Kaan:
 
 # Kazam
 Kazam:
-  regex: 'Kazam ([^;/)]+)( Build|\))|Trooper_X[0-9][0-9](?:[);/ ]|$)|Tornado 348'
+  regex: 'Kazam(?:[);/ ]|$)|Trooper_X[0-9][0-9](?:[);/ ]|$)|Tornado 348'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml:7122]: Does that change make sense here? Don't think `Kazam` might be the end of the user agent. Maybe that should work:

```suggestion
  regex: 'Kazam(?:[^);/]+)(?:[);/ ]|$)|Trooper_X[0-9][0-9](?:[);/ ]|$)|Tornado 348'
```

Guess it should be change in the model regex as well

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml]: I suggest replacing this rule with `Kazam(?:[^);/]+)(?:[);/ ]|$)|` the following  `Kazam[; _]|`  or `Kazam|`

Comment (sgiehl) [reply regexes/device/mobiles.yml]: that's fine. you can also optimize the model regex a few lines below

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml]: @sgiehl done

**Review by sgiehl — APPROVED** [review]

**Pull request merged**